### DISCUSSION TITLE
Typescript error for mution.mutate

### DISCUSSION THREAD

**Discussion by thanhtutzaw** [TanStack/query#5799] [Q&A]
I just got the ts error in mutate . I would like to update or delete data in UI . I already update or delete for firestore . But UI is not updating in first fetch . when It have cached , I  can get update with invalidateQueries .

![Screenshot (239)](https://github.com/TanStack/query/assets/71011043/ee941dd7-a659-45cf-b4b9-7cff706efb21)
![Screenshot (241)](https://github.com/TanStack/query/assets/71011043/8d3e2787-9cff-4e54-9573-268ed8886e34)

**Comment by TkDodo** [comment]
your `mutationFn` is untyped:

```diff
- mutationFn: async (pageParam) => ...
+ mutationFn: async (pageParam: string) => ...
```
